Modules/Sources/WordPressAppUI/RestApiUpgradePrompt.swift

```diff
@@ -0,0 +1,67 @@
+import SwiftUI
+import DesignSystem
+
+public struct RestApiUpgradePrompt: View {
+
+    private let localizedFeatureName: String
+    private var didTapGetStarted: () -> Void
+
+    public init(localizedFeatureName: String, didTapGetStarted: @escaping () -> Void) {
+        self.localizedFeatureName = localizedFeatureName
+        self.didTapGetStarted = didTapGetStarted
+    }
+
+    public var body: some View {
+        VStack {
+            let scrollView = ScrollView {
+                VStack(alignment: .leading) {
+                    Text(Strings.title)
```

Comment:
(nit) You could use `EmptyStateView` here.

Reply:
`EmptyStateView` content is centered on the screen. I don't think it's necessary to add a full-screen mode to it?

<img width="320" src="https://github.com/user-attachments/assets/1dd712af-3573-42aa-95ba-58d5d4e32ee1" />

Reply:
Looks like it also needs some work in terms of the layout 😞